types/react/index.d.ts

```diff
@@ -1979,6 +1979,7 @@ declare namespace React {
 
     interface DetailsHTMLAttributes<T> extends HTMLAttributes<T> {
         open?: boolean;
+        onToggle?: MouseEventHandler<T>;
```

Comment:
According to MDN [it should be a generic event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDetailsElement/toggle_event).

```suggestion
        onToggle?: ReactEventHandler<T>;
```

Do you have some docs reference that this indeed implements React's MouseEvent?

Reply:
Nope, you are absolutely right, it shouldn't be a mouse event, my bad